### DISCUSSION TITLE
Skeleton in DataTable

### DISCUSSION THREAD

**Discussion by modim30** [primefaces/community#1796] [PrimeReact] [answered]
I am trying to use skeleton as a prop in column body when the data is being loaded. But it shows the default empty message instead of rendering the skeleton, and after that the data is loaded. So instead of skeleton default empty message is shown.

Column code for reference :

{columns.map((col) => (
            <Column
              showFilterMenuOptions={false}
              showFilterOperator={false}
              filter={col.filter}
              filterPlaceholder="Search"
              hidden={col.hidden}
              sortable={col.sortable}
              key={col.field}
              field={col.field}
              header={col.headerName}
              exportable={!col.hidden}
              body={(rowData) => {
                return isloading ? <Skeleton /> : rowData[col.field];
              }}
            />
          ))}

**Comment by melloware** [comment, marked as answer]
Duplicate of this question and answer: https://github.com/orgs/primefaces/discussions/1128

**Reply by modim30** [reply]
But its for virtualScroller

**Reply by melloware** [reply]
That is the only way to accomplish what you want currently.

**Reply by modim30** [reply]
okay thanks for the clarification
